person_class.rb

```diff
@@ -0,0 +1,27 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(name = 'Unknown', age = nil, parent_permission: true)
+    @id = generate_id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    return false if @age.nil?
+
+    @age >= 18
+  end
```

Comment:
- [ ] according to the requirements, the of_age? method should be set as a private method.